lib/mutant/mutation.rb

```diff
@@ -40,11 +40,11 @@ def original_source
 
     # Test if mutation is killed by test reports
     #
-    # @param [Array<Report::Test>] test_reports
+    # @param [Result::Test] test_result
     #
     # @return [Boolean]
-    def self.success?(test_result)
-      self::TEST_PASS_SUCCESS.equal?(test_result.passed)
+    def success?(test_result)
```

Comment:
Why promote this to an instance method? It doesn't use any instance method state so should probably remain as a class method.